src/config.cpp

```diff
@@ -444,6 +444,7 @@ SimulationConfig::Input parseInputModule(const nlohmann::json& valueIt,
         parseMandatory(valueIt, "rise_time", debugStr, ret.riseTime);
         parseMandatory(valueIt, "decay_time", debugStr, ret.decayTime);
         parseOptional(valueIt, "random_seed", ret.randomSeed);
+        parseOptional(valueIt, "reversal", ret.reversal);
```

Comment:
can we make it clear what the default value is here, so it's not relying on the ctor?

Reply:
yes please, if the default is missing other places, fix that too :)